### PR TITLE
Judicial System / Added the formal request to overview screens.

### PR DESCRIPTION
**Pull request by addi**
Added the formal request to overview screens.

https://app.asana.com/0/1199153462262248/1198962953338955

## What

Adding more formality to the process.

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Screenshot 2020-11-16 at 13 37 50

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by gudjong** [comment]
Tvær athugasemdir varðandi orðalag í dómkröfunni:

- Nafn kærða ætti að vera í þágufalli í þessari setningu. Þetta er líka vandamál í PDF-inu og spurning hvort við getum ekki fengið þessari setningu breytt þanning að nefnifall passi.

- ".. á meðan gæsluvarðhaldinu stendur." hljómar ekki vel. Í PDF-inu notaði ég "... meðan á gæsluvarðhaldinu stendur." sem er réttara.

File: apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx [hunk 1]
```diff
@@ -49,30 +50,41 @@ export const JudgeOverview: React.FC = () => {
...
 
   useEffect(() => {
     document.title = 'Yfirlit kröfu - Réttarvörslugátt'
   }, [])
 
   useEffect(() => {
-    if (!workingCase && resCase) {
-      setWorkingCase(resCase)
+    if (!workingCase && data) {
+      setWorkingCase(data.case)
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:76]: Probably makes no difference, but data.case may be undefined even if data is not. Thus, we may be setting the working case to undefined.

File: apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx [hunk 2]
```diff
@@ -49,30 +50,41 @@ export const JudgeOverview: React.FC = () => {
...
     }
-  }, [workingCase, setWorkingCase, resCase])
+  }, [workingCase, setWorkingCase, data])
+
+  const hasIsolation = (rass: Case): boolean => {
+    const restrictions =
+      rass?.custodyRestrictions !== undefined
+        ? rass.custodyRestrictions
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:83]: I don't think we should use custodyRestrictions here. This should always be based on the request that came from the prosecutor regardless of what the judge decided to do later. This should be exactly the same as in the prosecutor overview.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx (hunk 2 above)
Comment (addi) [reply apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:83]: I see. I was thinking about the case when the judge had ruled and signed, and would only see the overview. But thats another story i guess.

Comment (addi) [reply apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:83]: Lol ups the variable name is rass.

**Review by gudjong — APPROVED** [review]

**Pull request merged**